app/javascript/plain-javascript/quadrant-well-failing.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
This seemed simple enough to not warrant a complete re-write of the well failing interface in vue. So plain javascript all the way.

This is a little more complicated than may appear necessary at first glance, due to the need to prevent users from clicking 'Select quadrant 1' and then unchecking some wells, without also unchecking the negative control.